specifyweb/frontend/js_src/lib/components/WbPlanView/Mapper.tsx

```diff
@@ -281,6 +282,18 @@ export function Mapper(props: {
     mappingPathIsComplete(state.mappingView) &&
     getMappedFieldsBind(state.mappingView).length === 0;
 
+  const handleDeleteLine = (line: number): void => {
```

Comment:
FYI: please use `function` keyword when function declaration requires `{ }` (i.e multiple statements) and `const` when function declaration does not require `{ }` (i.e single statement
OR enable eslint